samples/features/sql-big-data-cluster/private-bdc/scripts/deploy-private-bdc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+azdata bdc config init --source aks-dev-test --target private-bdc-aks --force
```

Comment:
You need to specify in the readme that az and azdata are pre-requisites and must be installed on the client machine, provide pointers to where to install from.